neurom/features/__init__.py

```diff
@@ -87,43 +99,72 @@ def _get_feature_value_and_func(feature_name, obj, **kwargs):
     # pylint: disable=too-many-branches
     is_obj_list = isinstance(obj, (list, tuple))
     if not isinstance(obj, (Neurite, Morphology, Population)) and not is_obj_list:
-        raise NeuroMError('Only Neurite, Morphology, Population or list, tuple of Neurite,'
-                          ' Morphology can be used for feature calculation')
+        raise NeuroMError(
+            "Only Neurite, Morphology, Population or list, tuple of Neurite, Morphology"
+            f"can be used for feature calculation. Got: {obj}"
+        )
 
     neurite_filter = is_type(kwargs.get('neurite_type', NeuriteType.all))
     res, feature_ = None, None
 
     if isinstance(obj, Neurite) or (is_obj_list and isinstance(obj[0], Neurite)):
+
         # input is a neurite or a list of neurites
         if feature_name in _NEURITE_FEATURES:
-            assert 'neurite_type' not in kwargs, 'Cant apply "neurite_type" arg to a neurite with' \
-                                                 ' a neurite feature'
+
+            assert 'neurite_type' not in kwargs, (
+                'Cant apply "neurite_type" arg to a neurite with a neurite feature'
+            )
+
             feature_ = _NEURITE_FEATURES[feature_name]
+
             if isinstance(obj, Neurite):
                 res = feature_(obj, **kwargs)
             else:
                 res = [feature_(s, **kwargs) for s in obj]
+
     elif isinstance(obj, Morphology):
+
         # input is a morphology
         if feature_name in _MORPHOLOGY_FEATURES:
+
             feature_ = _MORPHOLOGY_FEATURES[feature_name]
+
+            if "use_subtrees" in inspect.signature(feature_).parameters:
```

Comment:
sneaky, not super excited about it, but I suppose it works, and I can't think of something better

Can we at least factor it out, so the other places it's used can all be updated at once if we think of something better?